### DISCUSSION TITLE
A2M relation access

### DISCUSSION THREAD

**Discussion by alihassan161820** [directus/directus#10309] [Archive]
Is there a way to access m2a from the related collection ( a2m ) ?

**Comment by joselcvarela** [comment]
Hello @alihassan161820 
I believe this is not possible at the moment.
I am converting this into a Feature Request so it could be in our plans to add it on the future. 👍

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/9350
